code/src/components/List.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+import { Item } from './Item.js'
+import { AddTodo } from 'components/AddTodo'
+import { Summary } from 'components/Summary'
+import { Header } from 'components/Header'
+
+const Container = styled.div`
+    max-width: 400px;
+    margin-left: auto;
+    margin-right: auto;
+
```

Comment:
remove blank

Reply:
I think this looks good because it creates some space to the media query 😇